test/groovy/ArtifactSetVersionTest.groovy

```diff
@@ -1,19 +1,12 @@
 #!groovy
 import com.lesfurets.jenkins.unit.BasePipelineTest
-import com.sap.piper.DefaultValueCache
 import com.sap.piper.GitUtils
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.ExpectedException
 import org.junit.rules.RuleChain
-import util.JenkinsLoggingRule
-import util.JenkinsReadMavenPomRule
-import util.JenkinsShellCallRule
-import util.JenkinsWriteFileRule
-import util.JenkinsStepRule
-import util.JenkinsEnvironmentRule
-import util.Rules
+import util.*
```

Comment:
Wouln't it be better to keep the individual entries, since we can easier see what is really used?
Its also more some kind of refactoring and not directly related to the feature (... timezone).

Reply:
I'll change it back, this was automatically done by my IDE. I really don't care, managing import should be done by the IDE. What is your accepted rule for wildcard imports?